api/endpoints/asset.md

```diff
@@ -39,11 +39,10 @@ To update a Asset send JSON with updated value for one or more of the attributes
     
 + Request (application/json)
 
-            {
-                "model": {
-                    "id": 78
-                }
-            }
+			{
```

Comment:
might as well do this for all of the endpoints PUT calls (adding the id in the payload) - reservation, user, etc.

Reply:
@quesojean ✅ added those 👍